CHANGELOG.md

```diff
@@ -10,19 +10,19 @@ the code was deployed.
 
 ## [Unreleased]
 
-## [1.3.0] - 2020-12-22
-### Added
-- Endpoints
+### Added 2020-12-22
```

Comment:
Thanks for changing this. But the date should now be removed too. The dates are only for releases.